src/System.ComponentModel.TypeConverter/tests/BindingListTests.cs

```diff
@@ -0,0 +1,699 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information. 
+
+using System;
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+using System.Collections.Generic;
+using Xunit;
+using System.Reflection;
+
+namespace System.ComponentModel.Tests
+{
+	public class BindingListTest
+	{
+		[Fact]
+		public void BindingListDefaults ()
+		{
+			BindingList<string> l = new BindingList<string> ();
+			IBindingList ibl = (IBindingList)l;
+				
+			Assert.True (l.AllowEdit, "1");
+			Assert.False (l.AllowNew, "2");
+			Assert.True (l.AllowRemove, "3");
+			Assert.True (l.RaiseListChangedEvents, "4");
+
+			Assert.False (ibl.IsSorted, "5");
+			Assert.Equal (ibl.SortDirection, ListSortDirection.Ascending);
```

Comment:
Nit: in Assert.Equal, the first arg is the expected value and the second is the actual value, so these should be swapped